citesphere/src/main/java/edu/asu/diging/citesphere/web/user/AsyncCitationManager.java

```diff
@@ -8,6 +8,9 @@
 import java.util.concurrent.Future;
 
```

Comment:
I didn't notice that before but this class should not be in the controller package. It's a service class.